src/activities/binary_bulb/Binary_bulb.qml

```diff
@@ -0,0 +1,369 @@
+            property alias b4: b4
+            property alias b5: b5
+            property alias b6: b6
+            property alias b7: b7
+            property alias b8: b8
+        }
+
+        onStart: { Activity.start(items) }
+        onStop: { Activity.stop() }
+
+
+        DialogHelp {
+            id: dialogHelp
+            onClose: home()
+        }
+
+        Bar {
+            id: bar
+            content: BarEnumContent { value: help | home | level }
+            onHelpClicked: {
+                displayDialog(dialogHelp)
+            }
+            onPreviousLevelClicked: Activity.previousLevel()
+            onNextLevelClicked: Activity.nextLevel()
+            onHomeClicked: activity.home()
+        }
+
+        Bonus {
+            id: bonus
+            Component.onCompleted: win.connect(Activity.nextLevel)
+
+        }
+
+        IntroMessage {
+        	id: message
+        	onIntroDone : {
+        		Activity.initLevel
+        	}
+        	intro: [
+        		qsTr("Computers use Binary number system, where there are two symbols, 0 and 1."),
+        		qsTr("In decimal number system 123 is represented as 1 x 100 + 2 x 10 + 3 x 1"),
+        		qsTr("Binary represents numbers in the same pattern, but using powers of 2 instead of powers of 10 that decimal uses"),
+        		qsTr("So, 1 in binary is represented by 001, 4 by 100, 7 by 111 and so on.."),
+        		qsTr("Our computer has many many switches (called transistors) that can be turned on or off given electricity, and a switch that is on will represent a 1 and a switch that is off will represent a 0."),
+        		qsTr("In this activity, you are given a number, you have to find its binary representation by turning on the bubls. An on bulb representes 1 and an off bulb represents 0")
+        	]
+        	z: 20
+            anchors {
+           		top: parent.top
+            	topMargin: 10
+            	right: parent.right
+                rightMargin: 5
+                left: parent.left
+                leftMargin: 5
+            }        		
+        }
+
+	    property var sum: 0
+	    property var num: Math.floor(Math.random() * 255) + 1   
+
+	    function initialize_values()
+	    {
+	    	sum = 0
+	    	num = Math.floor(Math.random() * 255) + 1
+	    }
+
+        GCText {
+            id: question
+            anchors.horizontalCenter: parent.horizontalCenter
+            anchors.top: parent.top
+            anchors.topMargin: 10
+            fontSize: largeSize
+            width: parent.width * 0.9
+            horizontalAlignment: Text.AlignHCenter
+            wrapMode: Text.WordWrap
+            font.weight: Font.DemiBold
+            style: Text.Outline
+            styleColor: "black"
+            color: "white"
+            text: "What is the binary representation of " + num
+            opacity: 1.0
+        }
+
+
+	    Row {
+	    	id: row
+	    	anchors.centerIn: parent
+	    	spacing: 20
+
+	    	LightBulb {
+	    			id: b1
+	    			Text {
```

Comment:
GCText for texts to keep the same font and other common properties across all GCompris texts.